src/frontend/components/AppRoutes/index.tsx

```diff
@@ -1,59 +1,100 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
```

Comment:
In commit title
`acrchitecture` => `architecture`

In commit message
`This change also` => `This also changes`